.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/**
-target/**
\ No newline at end of file
+target/**
+utility-method.iml
```

Comment:
```suggestion
*.iml
```

Reply:
Other developers might have this .iml file but with different name, so it's better to use wildcard instead of a specific name